test/functional/tests/fault_injection/test_standby_activate_different_cache_path.py

```diff
@@ -0,0 +1,90 @@
+def test_standby_activate_different_cache_path(cache_line_size):
+    """
+        title: test_standby_activate_different_cache_path
+        description: |
+            Symlink-activated CAS partitions distinction test.
+        pass_criteria:
+          - Prepare two partitions with the same size, start and stop cache instance on one.
+          - Successfully start a standby cache with valid CAS metadata from the first partition.
+          - Activate cache providing symbolic link which points to second partition.
+          - Verify that the activation succeeded by checking length of array with caches.
+          - Verify that the regular cache instance is running by checking its state.
+    """
+    with TestRun.step("Prepare two partitions with the same size."):
+        cache = TestRun.disks["cache"]
+        cache.create_partitions([Size(200, Unit.MebiByte), Size(200, Unit.MebiByte)])
+        dev1 = cache.partitions[0]
+        dev2 = cache.partitions[1]
+
+    with TestRun.step("Start a regular cache instance on one of the partitions and stop it."):
+        cache = casadm.start_cache(dev1, force=True)
+        if cache.get_status() != CacheStatus.running:
+            TestRun.LOGGER.error("Failed to start cache.")
+        cache.stop()
+
+    with TestRun.step("Create a symbolic link to the second partition in ‘/dev/disk/by-id/`"):
+        cache2_dev_link = dev2.get_device_link("/dev/disk/by-id")
+
+    with TestRun.step("Start a standby instance on the second partition (but using the original path, not the symlink)"):
+        cls = int(cache_line_size.value.value / Unit.KibiByte.value)
+        standby_cache = casadm.standby_init(
+            cache_dev=dev2,
+            cache_id=1,
+            cache_line_size=cls,
+            force=True
+        )
+
+        if standby_cache.get_status() != CacheStatus.standby:
+            TestRun.LOGGER.error("Failed to activate standby instance on the second partition.")
+
+    with TestRun.step("Populate the standby cache with valid CAS metadata from the first partition"):
+        Dd().input(dev1).output(cache).run()
+
+    with TestRun.step("Detach standby cache"):
+        cache.standby_detach()
+        cache_status = cache.get_status()
+        if cache_status != CacheStatus.standby_detached:
+            TestRun.LOGGER.error("Failed to detach standby cache.")
+
+    with TestRun.step("Activate cache providing symlink"):
+        output = TestRun.executor.run(
+            standby_activate_cmd(
+                cache_dev=cache2_dev_link.full_path, cache_id=str(1),
+            )
+        )
+        if output.exit_code != 0:
+            TestRun.LOGGER.error("Failed to activate cache with provided symlink")
+
+    with TestRun.step("Verify that the activation succeeded, and the regular cache instance is running "):
+        caches = get_caches()
+        cache_status = caches[0].get_status()
+
+        if len(caches) < 1:
+            TestRun.LOGGER.error("Cache is not activated.")
```

Comment:
"Cache not present in system" more explicitly describes what happened in this check.

Reply:
I have changed the message.